handler/CMakeLists.txt

```diff
@@ -35,42 +34,68 @@ if(LINUX OR ANDROID)
 endif()
 
 if(LINUX)
-    target_sources(crashpad_handler PRIVATE
+    target_sources(crashpad_handler_lib PRIVATE
         linux/cros_crash_report_exception_handler.cc
         linux/cros_crash_report_exception_handler.h
     )
 endif()
 
 if(WIN32)
-    target_sources(crashpad_handler PRIVATE
+    target_sources(crashpad_handler_lib PRIVATE
         win/crash_report_exception_handler.cc
         win/crash_report_exception_handler.h
     )
 endif()
 
+target_link_libraries(crashpad_handler_lib
+    PRIVATE
+        $<BUILD_INTERFACE:crashpad_interface>
+    PUBLIC
+        crashpad_compat
```

Comment:
I just noticed that we drifted quite a bit from the deps inside the GN file. But if everything is working, I guess this is fine :-D